txpipe/twopoint_fourier.py

```diff
@@ -64,6 +64,9 @@ class TXTwoPointFourier(PipelineStage):
         "flip_g1": False,
         "flip_g2": False,
         "cache_dir": '',
+        "syst_nc": False,
```

Comment:
Could you give these a slightly more descriptive name, e.g. `deproject_clustering_sys`?  And also remove the lensing option for now as it doesn't do anything yet.

Reply:
ok